.git_archival.txt

```diff
@@ -0,0 +1,3 @@
+node: $Format:%H$
+node-date: $Format:%cI$
+describe-name: $Format:%(describe:tags=true,match=*[0-9]*)$
```

Comment:
We should probably adjust this match to something like `v[0-9]+.[0-9]+.[0-9]+`?

Reply:
Sure, it makes sense to be more specific. Although it looks like not all [tags](https://github.com/labgrid-project/labgrid/tags) follow that pattern. For example, major releases omit the patch number (e.g. `v24.0`). But I think `v[0-9]+.[0-9]+` should do the trick.

Reply:
It turns out `match` takes a [`glob(7)`](https://www.man7.org/linux/man-pages/man7/glob.7.html) pattern, not a regular expression. So I think it would be best to go for `v[0-9]*` for simplicity.